app/renderer/util.js

```diff
@@ -121,6 +137,119 @@ export function getOptimalXPath (doc, domNode, uniqueAttributes = ['id']) {
   }
 }
 
+/**
+ * Get an optimal Class Chain for a DOMNode based on the getOptimalXPath method
+ *
+ * @param {DOMDocument} doc
+ * @param {DOMNode} domNode
+ * @param {Array<String>} uniqueAttributes Attributes we know are unique
+ * @returns {string}
+ */
+function getOptimalClassChain (doc, domNode, uniqueAttributes) {
+  try {
+    // BASE CASE #1: If this isn't an element, we're above the root, or this is `XCUIElementTypeApplication`,
+    // which is not an official XCUITest element, return empty string
+    if (!domNode.tagName || domNode.nodeType !== 1 || domNode.tagName === 'XCUIElementTypeApplication') {
+      return '';
+    }
+
+    // BASE CASE #2: If this node has a unique class chain based on attributes then return it
+    for (let attrName of uniqueAttributes) {
+      const attrValue = domNode.getAttribute(attrName);
+      if (attrValue) {
+        let xpath = `//${domNode.tagName || '*'}[@${attrName}="${attrValue}"]`;
```

Comment:
I personally prefer using `const` everywhere for variables whose values are not going to be changed

Reply:
Fixed it, thanks